18F/identity-idp
# LG-6158: Use FullScreen component for modal component

**Why**: To reuse common accessibility behaviors, including inert background content, focus trap, autofocus, deactivation triggers (e.g. escape button).

**Testing Instructions:**

1. Set `idv_api_enabled: "true"` in local `config/application.yml`
2. Go to: http://localhost:3000/verify/v2/personal_key_confirm
3. Observe that input field has focus
4. Tab through content up to and including the end of content
5. Observe that focus loops back to beginning of content after end
6. Activate [screen reader web rotor](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)
7. Observe that the only content listed in web rotor is modal content

Extra credit: Test to confirm there are no regressions to document capture Acuant SDK full-screen dialog experience.

**Follow-up items:**

FullScreen `onRequestClose` behaviors should be wired up to return the user to the previous step on <kbd>Escape</kbd> keypress or on clicking the backdrop. This depends on proposed changes at #6219.
